specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json

```diff
@@ -4693,6 +4792,34 @@
           "description": "The operation error message."
         }
       }
+    },
+    "ExposureControlRequest": {
+      "type": "object",
+      "description": "The exposure control request.",
+      "properties": {
+        "featureName": {
+          "type": "string",
+          "description": "The feature name."
+        },
+        "featureType": {
+          "type": "string",
+          "description": "The feature type."
+        }
+      }
+    },
+    "ExposureControlResponse": {
```

Comment:
>ExposureControlResponse [](start = 5, length = 23)

Since this is a pure server response type and will never be sent back to server - Properties should be marked as readonly fields.

Reply:
Done.